any_parser/any_parser.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, api_key: str, base_url: str = PUBLIC_SHARED_BASE_URL) -> None
         self._headers = {
             "Content-Type": "application/json",
             "x-api-key": self._api_key,
+            "Authorization": "-"
```

Comment:
Do we really need this because there should be default Authorization value if it is not past in. We should not add unnecessary argument values unless it is mandatory.

Reply:
IC.